src/devtools/client/debugger/src/reducers/pause.ts

```diff
@@ -150,7 +150,7 @@ const pauseSlice = createSlice({
       Object.assign(state, {
         seekState: executionPoint ? "create-pause" : "find-point",
         id: undefined,
-        executionPoint,
+        executionPoint: executionPoint ?? null,
         time,
         pausePreviewLocation: location ?? null,
       });
```

Comment:
You could add a `satisfies` here

Reply:
<img width="823" alt="Screen Shot 2023-11-09 at 11 16 00 AM" src="https://github.com/replayio/devtools/assets/29597/1455a79b-b7d2-4a05-b371-b2efd1ea70bf">

Reply:
Unfortunately `satisfies Partial<PauseState>` won't catch the `undefined` (because that's always allowed for `Partial`). BUT it did show me another pretty stupid bug, so thanks for the suggestion!

Reply:
Yeah, if you say `satisfies PauseState` then it will catch the `undefined`... but then it complains that not all properties are defined (once you have fixed any other bugs).

Reply:
Yeah, that's an awkward gap. I guess we don't need to be using `Object.assign` here.